<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="PixelDisplay" name="PixelDisplay1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>55</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="updateCheckBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>with interval of [ms]:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="updateSpinBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_updMode">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>55</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_esc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Esc</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_left">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;&lt; / -</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_right">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>+ / &gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_logEnable">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionX1"/>
    <addaction name="actionX2"/>
    <addaction name="actionX3"/>
    <addaction name="actionX4"/>
    <addaction name="actionX5"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionX1">
   <property name="text">
    <string>x1</string>
   </property>
  </action>
  <action name="actionX2">
   <property name="text">
    <string>x2</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionX3">
   <property name="text">
    <string>x3</string>
   </property>
  </action>
  <action name="actionX4">
   <property name="text">
    <string>x4</string>
   </property>
  </action>
  <action name="actionX5">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>x5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PixelDisplay</class>
   <extends>QFrame</extends>
   <header>pixeldisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
